main.go

```diff
@@ -15,13 +15,30 @@
 package main
 
 import (
+	"log"
 	"os"
+	"runtime/pprof"
 
 	"github.com/ynqa/word-embedding/cmd"
 )
 
 func main() {
+
+	f, err := os.Create("test.prof")
+	if err != nil {
+		log.Fatal(err)
+	}
+	pprof.StartCPUProfile(f)
+	defer pprof.StopCPUProfile()
```

Comment:
I'd like to change these profiling code as optional.
So when only given flag (such as --prof), this code run.